.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!-- 
+Insert your contribution description here.
+Ensure your contribution fills the Contributing Guidelines described here:
+https://github.com/bonitasoft/bonita-doc/blob/7.3/CONTRIBUTING.md 
```

Comment:
This is the first version where we put the contributing guidelines, and they are the same on the branches above.